### ISSUE TITLE
Vehicle data is very US-centric

### ISSUE THREAD

**Issue by matthewmayer** [faker-js/faker#2881]
i noticed in #2874 that the vehicle data is heavily US-centric, mostly featuring US manufacturers and US models, whereas the generic en locale is supposed to cover all English-speaking regions. Once #2874 is merged i can follow up with a new PR to add some more popular missing brands popular in US, UK, India e.g.

* Subaru
* Mitsubishi
* Peugeot
* Citroën
* Suzuki
* Renault

Plus some EV manufacturers like 
* BYD
* NIO
* Rivian

**Comment by xDivisionByZerox** [comment]
I appreciate all the work you put into creating all these tickets with things you noticed during the data normalization PRs. Big thank you from my side 🙏
